src/Components/DnDLayout/GridLayout.tsx

```diff
@@ -383,7 +388,7 @@ const GridLayout = ({ isLayoutLocked = false }: { isLayoutLocked?: boolean }) =>
               </div>
             );
           })
-        }
+          .filter((layoutItem) => layoutItem !== null)}
```

Comment:
@Hyperkid123 I've filtered out the `null` elements here... not sure if this is what you meant? FWIW react-grid-layout seemed to handle the `null` elements in the active layout the same way react does (ignores them).

Or were you asking me to strip those elements from the `activeLayout` memo itself (pulls from jotai layout atom)?